props/saber_BC_buttons.h

```diff
@@ -581,9 +581,9 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
 
   void DetectMenuTurn() {
     float a = fusor.angle2() - current_menu_angle_;
+    if (a > M_PI) a-=M_PI*2;
```

Comment:
This code has nothing to do with sensitivity though, it controls the wrap-around.

Reply:
ok. it should still be the same for both modes.

Reply:
conflict resolved.